pam/integration-tests/integration_test.go

```diff
@@ -158,6 +157,7 @@ func TestCLIChangeAuthTok(t *testing.T) {
 			cmd := exec.Command("vhs", filepath.Join(currentDir, "testdata", "tapes", tc.tape+".tape"))
 			cmd.Env = testutils.AppendCovEnv(cmd.Env)
 			cmd.Env = append(cmd.Env, pathEnv)
+			cmd.Env = append(cmd.Env, strings.Join([]string{socketPathEnv, socketPath}, "="))
```

Comment:
Using fmt.Sprintf is less prone to error and offers more readability on what we are doing

```suggestion
			cmd.Env = append(cmd.Env, fmt.Sprintf("%s=%s", socketPathEnv, socketPath))
```